templates/iptables.rules.tmpl

```diff
@@ -21,4 +21,11 @@ COMMIT
 -A FORWARD -s {{.DockerSubnet}} ! -o docker0 -j ACCEPT
 -A FORWARD -o docker0 -j DOCKER
 -A OUTPUT -j ACCEPT
+{{else}}
+-A INPUT -i lo -j ACCEPT
+-A INPUT -d {{.Subnet}} ! -i docker0 -j ACCEPT
```

Comment:
I don't like this two docker0 related rules but I think we want them to be there.